meerkat_frontend/templates/reports/report_foreigner_screening.html

```diff
@@ -0,0 +1,83 @@
+{% extends 'reports/base.html' %}
+{% block title %}{{report.data.project_region|e}} &middot; Foreigner Screening  &middot; Week {{ report.data.epi_week_num }}{% endblock title %}
+{% block body %}
+<!-- CONTENT -->
+<div class="page-header">
+    <h2>
+	<span
+	    class="flag-icon flag-icon-{{report.flag}}"
+	    style="background-image: url(/static/img/flags/4x3/{{report.flag}}.svg) !important;"
+	>
+	</span>
+    <img class="moh-title-logo" src="/static/img/{{content['footer']['logos']['country_partner']}}">
+	  <span class="hidden-xs">{{report.data.project_region|e}} &middot; </span> Foreigner Screening Report
+    <span class="report-week">
+        {{ report.data.start_date|datetime_from_json|datetime('dd MMMM YYYY') }} - {{ report.data.end_date|datetime_from_json|datetime('dd MMMM YYYY') }}
+    </span>
+    </h2>
+
+</div>
+<div class="row">
+  <!-- Public Health Indicators -->
+  <div class="col-sm-12">
+	  <div class="chart-wrapper">
```

Comment:
Odd indentation? Looks like double indentation? I've started trying to stick to the usual 4 spaces.